src/app-layout/utils/use-focus-control.ts

```diff
@@ -9,7 +9,8 @@ export interface Focusable {
 export interface FocusControlRefs {
   toggle: RefObject<Focusable>;
   close: RefObject<Focusable>;
-  slider: RefObject<HTMLDivElement>;
+  slider: RefObject<Focusable>;
```

Comment:
why do we need to refs now? I see `slider` ref is used to focus on it inside Split panel. Can we instead use `handle` ref?

Reply:
I prepare the drag handle component to become a public component. Similar to other similar components, it does not give access to the dom node via ref but only allows to cast the focus programmatically - that is what the slider ref is used for.

The handle ref is a custom div wrapper around the drag handle to get the placement information from.